site/en/docs/devtools/network/reference/index.md

```diff
@@ -437,54 +394,48 @@ To view HTTP header data about a request:
 2.  Click the **Headers** tab.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/Zv6sbHi6fZfZMWufSp75.svg", alt="The Headers tab.", width="684", height="450" %}
-
-**Figure 23**. The Headers tab, outlined in blue
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/0l9iRxK3yvrnha53NiKH.png", alt="The Headers tab.", width="800", height="469" %}
 
 #### View HTTP header source {: #header-source }
 
-By default, the Headers tab shows header names alphabetically. To view the HTTP header names in the
+By default, the **Headers** tab shows header names alphabetically. To view the HTTP header names in the
 order they were received:
 
 1.  Open the **Headers** tab for the request you're interested in. See [View HTTP headers][15].
 2.  Click **view source**, next to the **Request Header** or **Response Header** section.
 
 
-#### Provisional headers are shown {: #provisional-headers }
-
-There are times where the headers will show a  warning message "Provisional headers are shown...".
-
-1. It could be due to the request not sent over the network (served from a local cache), which doesn't store the original request headers. In this case, you can [disable caching](#disable-cache) to see the full request headers.
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/AQfq69qS2Ig6lT9Y7vcc.png", alt="Provisional headers warning message", width="800", height="517" %}
+#### Provisional headers warning {: #provisional-headers }
 
-2. It could also be due to the network resource not valid (e.g. Try `fetch("https://jec.fyi.com/unknown-url/")` in the Console). DevTools could also display only provisional header due to security reasons.
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/xMKUHQxXLiRDiiUTol5J.png", alt="Provisional headers warning message", width="800", height="517" %}
+Sometimes the **Headers** tab shows the `Provisional headers are shown...` warning message. This may be due to the following reasons:
 
+- The request wasn't sent over the network but was served from a local cache, which doesn't store the original request headers. In this case, you can [disable caching](#disable-cache) to see the full request headers.
+  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/AQfq69qS2Ig6lT9Y7vcc.png", alt="Provisional headers warning message.", width="800", height="517" %}
 
-### View query string parameters {: #query-string }
+- The network resource isn't valid. For example, execute `fetch("https://jec.fyi.com/unknown-url/")` in the **Console**.
+  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/xMKUHQxXLiRDiiUTol5J.png", alt="Provisional headers warning message.", width="800", height="517" %}
 
-To view the query string parameters of a URL in a human-readable format:
+DevTools can also display only provisional headers due to security reasons.
 
-1.  Open the **Headers** tab for the request you're interested in. See [View HTTP headers][16].
-2.  Go to the **Query String Parameters** section.
+### View query string parameters (request payload) {: #query-string }
```

Comment:
There are two types of payload Query str params and Form Data. Probablt change the title to payload or add "and form data".

To debug, login Twitter, open network panel, refresh the page, filter request by fetch/xhr, and go through the list. There are both query str and form data example.